docs/source/transition.rst

```diff
@@ -0,0 +1,26 @@
+Transition to ``pydoctor``
+==========================
+
+From ``epydoc``
+---------------
+
+If you are looking for a successor to ``epydoc`` after moving to Python 3, ``pydoctor`` is the right tool for your project!
+
+.. important::
+
+    - ``pydoctor`` dropped support for the ``X{}`` tag. All other epytext markup syntax should be fully supported
+
+From ``pdoc3``
+--------------
+
+There a couple reasons to stop using ``pdoc3``, the main one beeing the swastika in their website footer.
```

Comment:
Maybe we should keep pydoctor documentation neutral and don't talk about swastika.

Just add a list of things to know for people who want to migrate from pdoc3 to pydoctor.

Each person can have its own reason to do the migration.

I think that if we keep mentioning the swastika think we just make it free publicity :)

Reply:
No free publicity.